tailwind.config.js

```diff
@@ -4,7 +4,20 @@ module.exports = {
     "./src/**/*.{js,jsx,ts,tsx}",
   ],
   theme: {
-    extend: {},
+    extend: {
+      transitionDuration: {
+        '900': '900ms',
+      },
+      colors:{
+        sbg: "#6949ff",
```

Comment:
Why don't you use the actual color names?
It would be descriptive for anyone seeing this in the future.